packages/xrpl/src/models/ledger/NFTokenPage.ts

```diff
@@ -1,21 +1,19 @@
 import BaseLedgerEntry from './BaseLedgerEntry'
 
 export interface NFToken {
-  Flags: number
-  Issuer: string
-  NFTokenID: string
-  NFTokenTaxon: number
-  URI?: string
-}
-
-export interface NFTokenWrapper {
-  NFToken: NFToken
+  NFToken: {
+    Flags: number
+    Issuer: string
+    NFTokenID: string
+    NFTokenTaxon: number
+    URI?: string
+  }
 }
 
 export interface NFTokenPage extends BaseLedgerEntry {
   LedgerEntryType: 'NFTokenPage'
   NextPageMin?: string
-  NFTokens: NFTokenWrapper[]
+  NFTokens: NFToken[]
```

Comment:
Why was this changed from `NFTokenWrapper`?

Reply:
To keep it inline with the other nested STObjects in STArrays.  After looking at more definitions i decided to change it.  NFTokenWrapper was never in a released version of xrpl.js so figured now would be a good time to change it.